packages/schematics/angular/library/files/__projectRoot__/ng-package.json

```diff
@@ -1,7 +1,6 @@
 {
   "$schema": "<%= relativePathToWorkspaceRoot %>/node_modules/ng-packagr/ng-package.schema.json",
   "dest": "<%= relativePathToWorkspaceRoot %>/<%= distRoot %>",
-  "deleteDestPath": false,
```

Comment:
I'd like to understand the motivation to remove this preset better. Does this option not play well with watch mode?

I originally added it to enable users to re-run build in dev mode without breaking watched files. But now with watch mode existing, this usage might be different.

I guess ideally we'd want a single `ng-package.json` file at all time, with the destination path being automatically deleted in single-run mode and not in watch mode.

But that does beg the question: if `deleteDestPath` does not interact well with watch mode, maybe it should be ignored in `ng-packagr` directly.

Reply:
Regardless of the final semantics of watch mode, I do agree on doing away with the dev/prod configs if their only difference is watch mode support.

Reply:
It's also important to remove the dist dir overall though. If you don't, artifacts that aren't there in subsequent builds will still be present because of previous builds.

With apps this is especially relevant because of assets, but less relevant for libraries. Still, the correct thing to do on a fresh, non-watch build is to clean (delete) the outpath before re-generating artifacts.

That was why we had it on on the production builds, and not the development ones. The production builds were the 'correct' ones, while the dev builds made that concession for a makeshift watch mode.

Reply:
By default `ng-packagr` deletes the `dist` directory when when triggering the build/watch

Reply:
Ah I wasn't aware of that! Then removing the option makes perfect sense.